src/asicverifier/src/test/resources/asic-containers/valid-signed-message/message.xml

```diff
@@ -1,25 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <SOAP-ENV:Envelope
-        xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"
-        xmlns:xroad="http://x-road.eu/xsd/xroad.xsd"
-        xmlns:id="http://x-road.eu/xsd/identifiers">
+    xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"
+    xmlns:id="http://x-road.eu/xsd/identifiers"
+    xmlns:xroad="http://x-road.eu/xsd/xroad.xsd">
     <SOAP-ENV:Header>
-        <xroad:client id:objectType="MEMBER">
-            <id:xRoadInstance>EE</id:xRoadInstance>
-            <id:memberClass>BUSINESS</id:memberClass>
-            <id:memberCode>consumer</id:memberCode>
+        <xroad:client id:objectType="SUBSYSTEM">
+            <id:xRoadInstance>DEV</id:xRoadInstance>
+            <id:memberClass>COM</id:memberClass>
+            <id:memberCode>4321</id:memberCode>
+            <id:subsystemCode>TestClient</id:subsystemCode>
         </xroad:client>
         <xroad:service id:objectType="SERVICE">
-            <id:xRoadInstance>EE</id:xRoadInstance>
-            <id:memberClass>BUSINESS</id:memberClass>
-            <id:memberCode>producer</id:memberCode>
-            <id:serviceCode>getState</id:serviceCode>
+            <id:xRoadInstance>DEV</id:xRoadInstance>
+            <id:memberClass>COM</id:memberClass>
+            <id:memberCode>1234</id:memberCode>
+            <id:subsystemCode>TestService</id:subsystemCode>
+            <id:serviceCode>getRandom</id:serviceCode>
+            <id:serviceVersion>v1</id:serviceVersion>
         </xroad:service>
-        <xroad:userId>EE:PIN:abc4567</xroad:userId>
-        <xroad:id>6289448e512292d17d54ef60ed2318e7</xroad:id>
-        <xroad:issue/>
+        <xroad:id>dd19fe68-7d37-41bd-b79c-8948ccdb36da</xroad:id>
+        <xroad:protocolVersion>4.0</xroad:protocolVersion>
     </SOAP-ENV:Header>
     <SOAP-ENV:Body>
-        <xroad:getState>aäõ</xroad:getState>
```

Comment:
I assume idea here was to check for UTF8 support, might not be a big issue that we loose it, but worth noting

Reply:
Message is never parsed while generating or vaildating asic containers, so we are probably ok. This looks like a standard query from other tests, so it should be covered where it matters.